### DISCUSSION TITLE
Error SendTextMessage - Newtonsoft.Json.JsonReaderException

### DISCUSSION THREAD

**Discussion by odiminformatica** [gabrieldwight/Whatsapp-Business-Cloud-Api-Net#47] [General]
I'm doing a test, and when calling the SendTextMessage method, it's giving the error below, any idea what it could be?

Newtonsoft.Json.JsonReaderException: 'Unexpected character encountered while parsing value: <. Path '', line 0, position 0.'

.NET 4.8 version
ConsoleApplication
Visual Studio 2017

Code snippet:

             WhatsAppBusinessCloudApiConfig whatsAppConfig = new WhatsAppBusinessCloudApiConfig();
             whatsAppConfig.WhatsAppBusinessPhoneNumberId = ConfigurationManager.AppSettings["WhatsAppBusinessPhoneNumberId"];
             whatsAppConfig.WhatsAppBusinessAccountId = ConfigurationManager.AppSettings["WhatsAppBusinessAccountId"];
             whatsAppConfig.WhatsAppBusinessId = ConfigurationManager.AppSettings["WhatsAppBusinessId"];
             whatsAppConfig.AccessToken = ConfigurationManager.AppSettings["AccessToken"];

             var httpClient = new HttpClient();
             httpClient.BaseAddress = WhatsAppBusinessRequestEndpoint.BaseAddress;
             var whatsAppBusinessClient = new WhatsAppBusinessClient(httpClient, whatsAppConfig);

             TextMessageRequest textMessageRequest = new TextMessageRequest();
             textMessageRequest.To = "5511975143038";
             textMessageRequest.Text = new WhatsAppText();
             textMessageRequest.Text.Body = "Message Body";
             textMessageRequest.Text.PreviewUrl = false;

             var result = whatsAppBusinessClient.SendTextMessage(textMessageRequest);

**Comment by gabrieldwight** [comment]
Hi, Is the error appearing when sending a message with the test phone number?

**Reply by odiminformatica** [reply]
yes

**Comment by odiminformatica** [comment]
Hello, yes, it is the test number reported by Meta.
<add key="WhatsAppBusinessPhoneNumberId" value="107792555563883" />

**Reply by gabrieldwight** [reply]
From the developer console for meta you need to send the hello world template message to your phone number and reply with any text to the test phone number then you will be able to send a message.
